src/fusor/fusor.py

```diff
@@ -543,11 +547,22 @@ def _sequence_location(
             else:
                 sequence_id = seq_id
 
+        # TODO: unsure if this is correct - I'm getting SQ's instead of SL's
```

Comment:
For SL identifiers, you should use `ga4gh_identify`. See last cell in this notebook: https://github.com/ga4gh/vrs-python/blob/main/notebooks/getting_started/3_Basic_Models.ipynb

Reply:
thank you so much! I'll get this added for the SequenceLocation

Reply:
jk, added a comment about this. It looks like we want to use the ga4gh digest for the SequenceLocation id here, but if I make a SequenceLocation object with this info just to get the id to make another one, doesn't that technically make it a new object and would end up changing the digest anyway? 

Or should I translate the identifier for `sequence_id` and use it instead? (It'll be a SQ instead of SL tho and in the examples we've discussed, the id has been SL)

Sorry for the confusion! Still trying to wrap my head around this :-D